src/components/AlignControl/index.tsx

```diff
@@ -0,0 +1,46 @@
+import { InfoCircleOutlined } from "@ant-design/icons";
+import { Radio, Tooltip } from "antd";
+import React from "react";
+import { createPortal } from "react-dom";
+
+const infoText = `\
+For the associated publication, single cell images were aligned before feature extraction. The \
+apical basal axis of the cell (which is the z-axis in the lab frame of reference) was preserved \
+and we aligned the cells by rotation in the xy-plane. Cells and nuclei are rotated such that the \
+longest cell axis falls along the x-axis.\
+`;
+
+type AlignControlProps = {
+    parent: HTMLElement;
+    aligned: boolean;
+    setAligned: (aligned: boolean) => void;
+};
+
+const AlignControl: React.FC<AlignControlProps> = ({ aligned, setAligned, parent }) =>
```

Comment:
I'm ok approving this as is but I still wish this were just a "Apply Transform" button in the viewer toolbar that was only being restyled and relabeled by CFE.

Reply:
This is certainly quite possible and would replace the hack I used to get the control into the toolbar with a marginally more limited and elegant hack, but would come at the cost of a bit of flexibility. In particular, I'm not certain there's a great way to add or change tooltip text without owning the component, so we might have to sacrifice that small bit of the specification. Personally, I could go either way: I think it would be nicer for the viewer to own this control, but can be persuaded that since so much of the specification is specific to this limited use case, it makes sense for it all to be defined at this level. I'd also be fine with merging this as-is now and setting an issue on the tracker to investigate moving responsibility down.

Reply:
At the risk of sounding ridiculous, props, and/or styles, are one way to replace the tooltip and relayout things inside a child component. (ridiculous because it scales out to allowing every button and label to be restyled) But yeah I wouldn't derail this PR for that kind of change unless it seemed like something would break.  Future work if at all...